src/sentry/api/endpoints/organization_member_issues_assigned.py

```diff
@@ -2,13 +2,16 @@
 
 from sentry.api.bases import OrganizationIssuesEndpoint
 from sentry.models import Group
+from django.db.models import Q
 
 
 class OrganizationMemberIssuesAssignedEndpoint(OrganizationIssuesEndpoint):
     def get_queryset(self, request, organization, member, project_list):
         return Group.objects.filter(
-            assignee_set__user=member.user,
-            assignee_set__project__in=project_list,
+            Q(assignee_set__user=member.user,
```

Comment:
This is totally fine assuming i understand the schema at this point. That said, you can slightly improve it by either:

1. remove the project constraint on the second query (team is already specific, isnt it?
2. change it to:

```
        .filter(
            Q(assignee_set__user=member.user,
                assignee_set__project__in=project_list) |
            Q(assignee_set__team__in=member.get_teams()),
            assignee_set__project__in=project_list
        )
```

Reply:
also if you want to be VERY sure its right, create more fixtures in the test to prove it doesnt include assignees that are invalid

Reply:
why does this have project__in twice? is that intentional?